tools/modules/docs/config.docs.sh

```
@@ -7,167 +7,102 @@ module_options+=(
 function generate_readme() {
 
-	# Get the current date
-	local current_date=$(date)
-	# setup doc folders
-	#mkdir -p "$script_dir/../share/doc/armbian-config"
+    # Get the current date
+    local current_date=$(date)
 
-	echo -e "Sorting data\nUpdating documentation" # current_date ;
+    # Setup the documentation and man-page directories
+    local doc_dir="$script_dir/../"
```

Shellcheck (0.10.0)

[warning] script_dir is referenced but not assigned.

(SC2154)